BMM.Core/Translation/en/main.json

```diff
@@ -246,7 +246,9 @@
     "Minutes": "{0} minutes",
     "Hour": "{0} hour",
     "Hours": "{0} hours",
-    "Disable": "Disable"
+    "Disable": "Disable",
```

Comment:
What's the disable used for?

Reply:
It's used for "sleep timer" functionality, in this PR I only added a comma at the end.